calico_kubernetes/logutils.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 
-LOG_DIR = '/var/log/calico/kubernetes'
+LOG_DIR = '/var/log/calico/kubernetes/exec-plugin/'
```

Comment:
Why have you added another directory? Does kubernetes/ have anything in it except this log? I'd be inclined to keep it as it is (or even use /var/log/calico/kubernetes-plugin.log instead of /kubernetes/calico.log)